apps/judicial-system/backend/src/app/modules/notification/defendantNotification.service.ts

```
@@ -0,0 +1,170 @@
+    theCase: Case,
+  ): Promise<DeliverResponse> {
+    {
+      switch (notificationType) {
+        case DefendantNotificationType.DEFENDER_ASSIGNED:
+          return this.sendDefenderAssignedNotification(defendant, theCase)
+        default:
+          throw new InternalServerErrorException(
+            `Invalid notification type: ${notificationType}`,
+          )
+      }
+    }
```

Biome

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)